code/game/data_huds.dm

```diff
@@ -398,6 +398,27 @@
 	var/amount = min(round(sunder * 100 / xeno_caste.sunder_max, 10), 100)
 	holder.icon_state = "sundering[amount]"
 
+/mob/living/proc/hud_set_firestacks()
```

Comment:
Need autodoc

Reply:
what does this mean

Reply:
Check dev general pins

It is commenting new procs and variable
In your case it would be /// Set firestacks on the hud
/mob/living/proc/hud_set_firestack()